src/native/corehost/bundle/reader.h

```diff
@@ -100,14 +100,10 @@ namespace bundle
             m_ptr += len;
         }
 
-        // Return a pointer to the requested bytes within the memory-mapped file.
-        // Skip over len bytes.
-        const char* read_direct(int64_t len)
+        uint8_t read_byte()
```

Comment:
Nit: since this is only used from one place, is it worth adding? Why not just use the `read` even for the single byte file type?

Reply:
Just noticed the same thing - there is `read()`, which is the same as `read_byte()`